CONTRIBUTING.md

```diff
@@ -213,10 +213,9 @@ Make sure that no system files (like `.DS_Store`) are in the manifest and add it
 contribution.
 
 ## Code Style
-MetPy uses the Python code style outlined in `PEP8
-<https://pep8.org>`_. For better or worse, this is what the majority
-of the Python world uses. The one deviation is that line length limit is 95 characters. 80 is a
-good target, but some times longer lines are needed.
+MetPy uses the Python code style outlined in [PEP8](http://pep8.org). For better or worse, this 
```

Comment:
Can you update this to use https? It eliminates a redirect if you do so.

Reply:
:+1: sorry about that.